corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -202,11 +202,17 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
             }
         ">
     <label class="caption control-label" data-bind="css: labelWidth, attr: {'for': entry.entryId}">
-      <div>
+      <div data-bind="css: {'row': help()}">
+        <span class="webapp-markdown-output" tabindex="0"
+              data-bind="
+                html: ko.utils.unwrapObservable($data.caption_markdown) || caption(),
+                css: {'col-md-11' : help()}"></span>
         <!-- ko if: help() -->
         <a
-          class="help-text-trigger"
+          class="help-text-trigger col-md-1"
           role="button"
+          href="javascript:void(0)"
```

Comment:
this line just shows the dialog without refreshing the page, right? Same this as if we said something like `onclick="return false";` ?

Reply:
Yep, without the href attribute this link was not getting focused on <tab> key, so using `href="javascript:void(0)"` it gets focused and causes no redirect/refresh on click.